purpose.md

```diff
@@ -0,0 +1,23 @@
+# Our Purpose 🌱
+
+## Preamble 
+The purpose serves to constitute Radworks, memorializing what we believe and the organization’s raison d’etre. It’s intended as a living document: a North Star that the community can return to for clarity in times of change, challenge, confusion, or conflict. If successful, this document will help protect and further Radwork’s purpose over time as we collectively navigate change. 
```

Comment:
Change It's to Its in 2nd sentence

Reply:
I don't think its would be grammatically correct, the sentence reads as "it is intended..." so I think it's is appropriate.